fuse.py

```diff
@@ -26,6 +26,8 @@
 
 import logging
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
That is a good point. I added it because the examples weren't printing out anything by default. But with that in mind, this should be added to the examples directly. I will take some time to instead move this as boilerplate to all the examples.